docker-compose/Makefile

```diff
@@ -49,6 +52,17 @@ download:
 	     echo "Skipping downloading the code from git as PROJECT_HOME is not default:" $(PROJECT_HOME); \
 	fi
 
+.PHONY: download-catalog
+download-catalog:
+	rm -rf ./openwhisk-catalog*
+	if [ "$(CATALOG_HOME)" = "./openwhisk-catalog" ]; then \
+	    curl -O ./openwhisk-catalog.tar.gz -L https://api.github.com/repos/apache/incubator-openwhisk-catalog/tarball/master > ./openwhisk-catalog.tar.gz; \
```

Comment:
Should probably move away from these api.github links for downloading stuff, they seem to report curl errors, see #85 and its fix.